aries-site/src/pages/components/button.mdx

```diff
@@ -62,6 +64,26 @@ The HTML elements for buttons and links (a.k.a. anchors) describe a very specifi
 - Use buttons when you are performing an action, such as: "submit," "merge," "create new," "upload," etc.
 - An action almost always occurs on the same page
 
+### Cancel Buttons
+
+Cancel buttons enable users to exit out of or dismiss the current screen in order to go back to the previous one. Having an explicit way to cancel provides a fallback to safety as it helps discard any unwanted changes. 
+
+#### Styling
+Visually emphasizing the cancel button should be avoided to minimize user confusion. The cancel button should appear neutral to signify the non-committal action. For this reason, cancel buttons should be presented in the Default Button styling.
```

Comment:
```suggestion
Cancel buttons should be presented in the default button styling. The neutral appearance signifies the non-committal action. Visually emphasizing the cancel button should be avoided to minimize user confusion.
```
Leading with the action we want people to take (use default button) and following it up with the reasoning.